src/app/business/profile/profileCard/profile-card.component.ts

```diff
@@ -104,12 +104,7 @@ export class ProfileCardComponent implements OnInit {
         private confirmationService:ConfirmationService
     ) { }
     option = {};
-    pools = [];
-    totalFreeCapacity = 0;
```

Comment:
just freeCapacity ? (instead of totalFreeCapacity)

Reply:
@rhsakarpos I did not quite understand what you meant here.  
The code is removed and not added.

Reply:
just rename the variable..the total is redundant imho..LG...AB